crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -989,15 +989,15 @@ pub struct RoomInfo {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub(crate) cached_display_name: Option<DisplayName>,
 
-    /// The recency timestamp of this room.
+    /// The recency stamp of this room.
     ///
     /// It's not to be confused with `origin_server_ts` of the latest event.
     /// Sliding Sync might "ignore” some events when computing the recency
-    /// timestamp of the room. Thus, using this `recency_timestamp` value is
+    /// stamp of the room. Thus, using this `recency_stamp` value is
     /// more accurate than relying on the latest event.
     #[cfg(feature = "experimental-sliding-sync")]
     #[serde(default)]
-    pub(crate) recency_timestamp: Option<MilliSecondsSinceUnixEpoch>,
+    pub(crate) recency_stamp: Option<u64>,
```

Comment:
I guess we don't need a serde `alias`here because the types have different semantic meaning?

Reply:
Exactly